include/abb_librws/v1_0/rw/cfg.h

```diff
@@ -0,0 +1,28 @@
+
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+#include <iosfwd>
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: cfg
+{
+    /**
+     * \brief A function for loading configs to robot controller.
+     *
+     * https://developercenter.robotstudio.com/api/rwsApi/cfg_system_load_page.html
+     *
+     * \param client RWS client
+     * \param resource specifies the RAPID task, module, and symbol name for the RAPID resource.
```

Comment:
Here it specifies a file, not the RAPID task, module, and symbol name. Please correct the comment.

Reply:
:+1: